Automattic/Gridicons-iOS
# Fixes filter icon

To test:

- Run the demo app
- verify that the filter icon is centered and occupies the full area of the icon
